download-temporal-cli.js

```diff
@@ -0,0 +1,59 @@
+import { join } from 'path';
```

Comment:
Should we put this in a `/bin`, or `/scripts` directory?

Reply:
`/bin` is ignored, so not there. Maybe if we ever have more than one, we can move it into a `scripts` directory.